userbenchmark/dynamo/common.py

```diff
@@ -257,6 +257,7 @@ class CI(NamedTuple):
     *CI_SKIP[CI("aot_eager", training=False, dynamic=True)],
     *CI_SKIP[CI("inductor", training=False)],
     "nanogpt_generate",  # Assertion `index out of bounds: 0 <= tmp0 < 64` failed.
```

Comment:
I guess we need to remove `nanogpt_generate` after https://github.com/pytorch/benchmark/pull/1911 lands?

Reply:
that's right, I will remove it in #1911